Fix several desktop icons

Mno-hime opened the pull request:
* XTerm

`Icon=terminal` seems to be a better match, which works for all icon
themes we ship (tested that), otherwise stub icon is used.

Disabled tests as I could not find any test target in Makefile.in.

* XScreensaver

We don't ship `Icon=gnome-ccscreensaver` outside Nimbus theme, replacing
with the same icon under a different name.

* system-config-printer

We don't ship `Icon=print-manager` outside Nimbus, replacing with
similar one.

alarcher commented:
@Mno-hime I had to revert the ove to 64-bit that I had done ealier. There are issues with terminal definitions and this will not work as expected.

Mno-hime committed:
Fix several desktop icons

* XTerm

`Icon=terminal` seems to be a better match, which works for all icon
themes we ship (tested that), otherwise stub icon is used.

Disabled tests as I could not find any test target in Makefile.in.

* XScreensaver

We don't ship `Icon=gnome-ccscreensaver` outside Nimbus theme, replacing
with the same icon under a different name.

* system-config-printer

We don't ship `Icon=print-manager` outside Nimbus, replacing with
simmilar one.

Mno-hime commented:
@alarcher I see, will stick to the `Icon` fix then.
